autoptsserver.py

```diff
@@ -224,18 +224,17 @@ def _dispatch_nonblocking(self, method_name, *args):
         self.ptscontrol_request_queue.put((method_name, False, args))
         return "WAIT"
 
-
     def _dispatch_blocking(self, method_name, *args):
         self.ptscontrol_request_queue.put((method_name, True, args))
-        result = None
         while not self._end.is_set():
+            result = None
             try:
                 result = self.ptscontrol_response_queue.get(block=True, timeout=1)
+            except Empty:
+                continue
+            else:
                 self.ptscontrol_response_queue.task_done()
                 return result
-            except Empty:
-                pass
-
         return result
```

Comment:
Hmm, wouldn't it have been easier to just add `return None` to the `except` clause? 

Interestingly, in the current (non-PR) version, line 239 would also always return None, as any non-failure path would return on line 235